### ISSUE TITLE
Can't use latest lombok (1.16.6)

### ISSUE THREAD

**Issue by tterrag1098** [franzbecker/gradle-lombok#6]
I calculated the sha256, it still fails on verifyLombok:

```
:verifyLombok FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':verifyLombok'.
> Could not find 'lombok-1.16.6.jar' in dependencies of configuration 'lombok'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

My code:

``` groovy
lombok {
    version = "1.16.6"
    sha256 = "e0a471be03e1e6b02bf019480cec7a3ac9801702bf7bf62f15d077ad4df8dd5d"
}
```

**Comment by tterrag1098** [comment]
Just occurred to me that you might want the stacktrace as well: http://puu.sh/kO2WR.txt

**Comment by franzbecker** [comment]
Thanks for reporting that, I could reproduce and fix it.
I released a new version of the plugin: 1.4.

**Comment by tterrag1098** [comment]
Thanks for the quick fix! working great now :+1:
